internal/util/util.go

```diff
@@ -8,3 +12,23 @@ func ToPtrSlice[T any](slice []T) []*T {
 	}
 	return result
 }
+
+func ToPtr[T any](x T) *T {
+	return &x
+}
+
+// SliceToCommaSeparated converts a string slice to a comma-separated string.
+func SliceToCommaSeparated(slice []string) string {
```

Comment:
Would it be simpler for the below code to use `strings.Join()` on single-quoted values? Maybe something like
```
       mySlice := []string{"dejan", "rich", "dave", "denis"}
        quoted := []string{}
        for _, s := range mySlice {
                quoted = append(quoted, "'"+s+"'")
        }
        commastr := strings.Join(quoted, ",")
```

Reply:
added 2 modes, one with quotes and one without. In SQL we need them quoted.